content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -1,215 +1,146 @@
-files. An indication of missing files in any of the databases may result
-from either of the following:
-
-- The recovery path of the nodes has not been set appropriately
-- There is some issue with the persistence files (for example: lack of
-    permissions to read the files, missing files, or corrupted files)
-
-First, make sure that the recovery path is set correctly on all of the
-nodes in the cluster. If that does not resolve the issues, [contact our
-support team](mailto:support@redislabs.com).
-
-To recover all of the databases, run the command `rladmin recover all`.
-
-To recover the databases one by one, run the command `rladmin recover db <id|name>`.
-
-After recovering the databases, you can run `rladmin status`
-to verify that the recovered databases are now active.
-
-**Note**: If you are running the recovery process on the same machines
-that were used for the original cluster and there are still Redis
-processes running on the machine the database recovery process will
-fail. Make sure to kill all Redis processes before attempting to run the
-recovery process.
+    database persistence files.
+
+    {{% note %}}
+Make sure that the user redislabs has permissions to access the storage location
+of the configuration and persistence files on each of the nodes.
+    {{% /note %}}
+
+    If you use local persistent storage, place all the recovery files on each of the cluster nodes.
+
+1. To recover the cluster with the configuration from the original cluster,
+    from the [rladmin]({{< relref "/rs/references/cli-reference/rladmin.md" >}}) command-line interface (CLI):
+
+    ```src
+    cluster recover filename [ <persistent_path> | <ephemeral_path> ]<filename> node_uid <node_uid> rack_id <rack_id>
+    ```
+
+    {{% expand "Command syntax" %}}
+`<filename>` - The full path of the old cluster configuration file in the persistent storage.
+The cluster configuration file is /css/ccs-redis.rdb.
+The file exists on the persistent storage drive of all nodes and all copies are identical.
+
+`<node_uid>` - The id of the node, in this case `1`.
+
+`<persistent_path>` (optional) - The location of the [persistent storage ]
+({{< relref "/rs/administering/designing-production/persistent-ephemeral-storage.md" >}})
+in the new node.
+
+`<ephemeral_path>` (optional) - The location of the [ephemeral storage]
+({{< relref "/rs/administering/designing-production/persistent-ephemeral-storage.md" >}})
+in the new node.
+
+`<rack_id>` - optional. If [rack-zone awareness]({{< relref "/rs/concepts/high-availability/rack-zone-awareness.md" >}})
+was enabled in the cluster,
+you can use this parameter to override the rack ID value that was
+set for the node with ID 1, with a new rack ID. Otherwise, the node
+will get the same rack ID as the original node.
+    {{% /expand %}}
+
+    For example:
+
+    ```src
+    rladmin cluster recover filename /tmp/persist/ccs/ccs-redis.rdb node_uid 1 rack_id 5
+    ```
+
+    When the recovery succeeds,
```

Comment:
Recovery process was just initiated. When this phase is done, the recovery isn't completed yet.

Reply:
How's this?

> When the recovery command succeeds,